packages/terra-tag/src/index.js

```diff
@@ -0,0 +1,7 @@
+import TagList from './TagList';
+import RollupTag from './RollupTag';
+import TagsUtils from './TagsUtils';
+import Tag from './Tag';
+
+export default Tag;
+export { TagList, RollupTag, TagsUtils };
```

Comment:
I thought only `TagList` was exported. Are `RollupTag` and `TagsUtils` also meant for consumers?
cc: @supreethmr

Reply:
Changes Done. Thanks

Reply:
+1